README.md

```diff
@@ -1,28 +1,32 @@
-# MicroflowTimer [![Support](https://img.shields.io/badge/Mendix%20Support%3A-Platform-green.svg)](https://docs.mendix.com/community/app-store/app-store-content-support)
+# ActionTimer [![Support](https://img.shields.io/badge/Mendix%20Support%3A-Platform-green.svg)](https://docs.mendix.com/community/app-store/app-store-content-support)
 
-This widget can be used to time and execute a Microflow as long as a certain form is open. The microflow can be executed once or repeatedly, in which case it will not stop until it returns false or until the form is closed.
+This widget can be used to time and execute a Microflow as long as a certain form is open. The microflow or nanoflow can be executed once or repeatedly, in which case it will not stop until it returns false or until the form is closed.
 
 ## Contributing
+
 For more information on contributing to this repository visit [Contributing to a GitHub repository](https://world.mendix.com/display/howto50/Contributing+to+a+GitHub+repository)!
 
-##Typical usage scenario
+## Typical usage scenario
 
 * Update a grid or object after a certain amount of time, for example a message inbox
 * Close a form with a message automatically after 10 seconds
 * Automatically make backup copies while the user is still editing an object.
 * Open a form and directly trigger validation errors. (to achieve this, use interval: 0, execute once: true, start at once: true) (new in 1.2)
 
-##Features and limitations
+## Features and limitations
+
+* Adds timed behavior to your applications.
+* Nanoflow support starts from Mendix 7.13.1.
```

Comment:
Run a Microflow or Nanoflow

Reply:
no need to mention version number as it will not be compatible < 7.13.1

Reply:
done